frontend/src/components/Modal/ModalAddBar.tsx

```diff
@@ -11,7 +11,7 @@ type AddBarProps = {
 
 export default function ModalAddBar(props: AddBarProps) {
 
-    const [bar ,setBar] = useState(props.bar)
+    const [ ,setBar] = useState(props.bar)
```

Comment:
```suggestion
    const [ bar,setBar] = useState(props.bar)
```
Aber warum musst du einen extra State davon erstellen, denn gibt es doch schon woanders?